tests/lib/pkgdb.sh

```diff
@@ -659,7 +659,9 @@ pkg_dependencies_ubuntu_classic(){
                 eatmydata
                 evolution-data-server
                 fwupd
+                gcc-multilib
```

Comment:
We list this as Build dependencies in debian/control for Ubuntu. I think we should do the same on sid, otherwise the unit tests will during build unless one sets `nocheck`